openedx_webhooks/jira/tasks/update_latest_github_activity.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not sure the "jira/tasks" hierarchy is helping us here.  How is this task more about jira than it is about github? It has jira in the directory and github in the file name.  This code might be over-hierarchialized.